### ISSUE TITLE
Spell check correction does not trigger a change event

### ISSUE THREAD

**Issue by edfxphn** [ianstormtaylor/slate#4055]
#### Do you want to request a _feature_ or report a _bug_?

I would say this is a bug (but may be an incorrect usage).

#### What's the current behavior?

I am new to Slate. I implemented it with an autosave feature. I searched in the existing issues, but do not find the same issue.

When fixing an error through  the spell check of Firefox, the word is fixed, but the "onChange" event is not triggered. If I type a new key, the onChange is triggered, and the error is backed again.

I tried with autoCorrect, autoCorrect="true", ... but nothings working.

This behavior is reproductible in the samples.

![Example](http://g.recordit.co/4rodUkRtDM.gif)

Slate: 0.59.0
Browser: Firefox 84.0.2
OS: macOS 11.1

#### What's the expected behavior?

Errors are fixed in the underlying model, and change event is triggered.

**Comment by BrentFarese** [comment]
How does Firefox spellcheck work?  Does it just update the DOM?  If Slate's model is not updated, `onChange` will not fire.

**Comment by edfxphn** [comment]
I don't know. It seems only a "input" event is fired which is ignored by Slate.
PS: it's not specific to Firefox ; it is the same with all navigators.

**Comment by stefanoverna** [comment]
having the same issue here. happy to contribute with a PR if someone can point me in the right direction!

**Comment by mattwalters** [comment]
I'm having the same problem. For some reason this seems to have only started for us in the last few days. Does anyone have any insight here. Echoing @stefanoverna here... happy to contribute a PR if someone can point me in the right direction.

**Comment by mdmag** [comment]
I have the same issue.

**Comment by mattwalters** [comment]
@mdmag FWIW, Updating to the latest version solved my problem.

**Comment by mdmag** [comment]
@mattwalters, I will update from version 0.57. 
So, I assume that there are a lot of breaking changes?

**Comment by mcturner1995** [comment]
Having the same issue. Has anyone found a solution?

**Comment by mattwalters** [comment]
@mcturner1995 What version are you using?

**Comment by mcturner1995** [comment]
Here are my versions from my package.json:
```
    "slate": "^0.59.0",
    "slate-history": "^0.59.0",
    "slate-hyperscript": "^0.59.0",
    "slate-react": "^0.59.0",
```

**Comment by dylans** [comment]
This was fixed a long time ago, but well after 0.59.0 (which is now more than 2 years old... we've had roughly 100 releases since then). You probably need 0.72.8 or later but you probably would just want to go all the way for the latest (0.84.0) if updating from something that old.
